docs/html/cli/pip_install.rst

```diff
@@ -205,8 +205,12 @@ Per-requirement Overrides
 -------------------------
 
 Since version 7.0 pip supports controlling the command line options given to
-``setup.py`` via requirements files. This disables the use of wheels (cached or
-otherwise) for that package, as ``setup.py`` does not exist for wheels.
+``setup.py`` via requirements files.
+
+.. warning::
+
+   This disables the use of wheels (cached or otherwise),
+   as ``setup.py`` does not exist for wheels.
```

Comment:
The reason is more complex than that. It's because `--install-options` does not make sense for `setup.py bdist_wheel`. But `--global-options` does, but we don't support it... so... it's complicated. It might be better to not say why at all...

Reply:
I've updated this to remove the "...as ``setup.py`` does not exist for wheels" part of the warning.